fedmsg/api.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of fedmsg.
+# Copyright (C) 2017 Red Hat, Inc.
```

Comment:
Since you used (C) instead of ©, do you need the utf-8 declaration?

Reply:
I would say that declaring encoding is always a good thing, even if it's not strictly necessary at the time the file is first written :-)